shared/general/labels.md

```diff
@@ -1,6 +1,6 @@
 Label | Default | Description | Supervisor | balenaOS*
 --- | --- | --- | --- | ---
-io.{{ $names.company.short }}.features.balena-socket | false | Bind mounts the balena container engine socket into the container. | v7.23.0 | v2.21.0
+io.{{ $names.company.short }}.features.balena-socket | false | Bind mounts the balena container engine socket into the container under `/host/run/balena-engine.sock`. | v7.23.0 | v2.21.0
```

Comment:
```suggestion
io.{{ $names.company.short }}.features.balena-socket | false | Bind mounts the balena container engine socket into the container and sets the environment variable `DOCKER_HOST` with the socket location for use by docker clients. | v7.23.0 | v2.21.0
```

Reply:
@ab77 I had forgotten to add this suggestion, I think you have to rebase the master branch as well in order for the changes to be merged. I can do it if you want.

Reply:
@pipex please, if you don't mind to take over this PR..